### ISSUE TITLE
Block fake email addresses

### ISSUE THREAD

**Issue by jamesgeddes** [GeekZoneHQ/web#185]
Check if user is attempting to register with a fake email address such as mailinator.com

Use an API such as https://www.block-disposable-email.com/ to check email domain. If domain found, return the following error

> We get it, spam is mega annoying and using a disposable email address is a great way to avoid it. Geek.Zone is all about real relationships with real people, so we need to be able to contact you. We promise not to spam you! Please use your real email address to register with us.

**Comment by CarwynNelson** [comment]
I think google allows things like `james+twitter@geek.zone` which becomes just `james@geek.zone`, so maybe we could suggest those sorts of things to people?

Much like my `check*` email addresses where if my email for a service gets leaked I can just block incoming email from `checktwitter` for example and then set a new email address for the service.

Does that make sense?

**Comment by jamesgeddes** [comment]
Yes, that is something we can suggest as an alternative, absolutely. This ticket is to prevent people from being able to register using james@mailinator.com for example, as we need real people with real addresses

**Comment by jamesgeddes** [comment]
The following might be helpful for this.
- [django-disposable-email-checker](https://pypi.org/project/django-disposable-email-checker/)
- [block-disposable-email](https://pypi.org/project/block-disposable-email/)

**Comment by jamesgeddes** [comment]
@NoAnswerss Are you still looking at this? I know you are more focused on the front-end stuff, so are you happy for me to reassign this?

**Comment by NoAnswerss** [comment]
Hello,

I am no lonnger on this email blocker, yes reassign it please 😊
